airflow/docker.go

```diff
@@ -275,36 +270,31 @@ func (d *DockerCompose) Start(imageName, settingsFile, composeFile, buildSecretS
 	if err != nil {
 		return errors.Wrap(err, composeRecreateErrMsg)
 	}
-	var airflowMessage string
-	if CheckM1Image(imageLabels) {
-		airflowMessage = " This might take a few minutes…"
-	} else {
-		airflowMessage = ""
-	}
 
-	fmt.Println("\n\nAirflow is starting up!" + airflowMessage)
+	fmt.Println("\n\nAirflow is starting up!")
 
 	airflowDockerVersion, err := d.checkAiflowVersion()
 	if err != nil {
 		return err
 	}
 
-	startupTimeout = waitTime
-	// check if user provided a waitTime
-	// default is 1 minute
-	if waitTime != 1*time.Minute {
-		startupTimeout = waitTime
-	} else if CheckM1Image(imageLabels) {
-		// user did not provide a waitTime
-		// if running darwin/M1 architecture
-		// we wait for a longer startup time
-		startupTimeout = 5 * time.Minute
```

Comment:
I'm removing a chunk of logic around this M1 architecture check and runtime comparison. The only reason we have all this is to bump the startup timeout from `1m` to `5m`. I've just made `5m` the default across the board.

Reply:
If we feel strongly about _not_ changing the default across the board, we could revert this piece. It just feels like unnecessary cruft at this point, and this feels like a fine time to remove it.

Reply:
Yea, I'm fine with making it 1m across the board now. @neel-astro that sound ok to you?